helpers.go

```diff
@@ -38,3 +39,11 @@ func (sc *Scanner) hasPackage(pkg, name string) bool {
 	}
 	return false
 }
+
+func (sc *Scanner) FileContains(path, content string) bool {
```

Comment:
__[gometalinter]__
* [golint] exported method Scanner.FileContains should have comment or be unexported